common/build.gradle

```diff
@@ -179,13 +179,15 @@ dependencies {
     testImplementation "org.json:json:$rootProject.ext.jsonVersion"
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
+    testImplementation "org.mockito.kotlin:mockito-kotlin:3.6.28"
     testImplementation "org.powermock:powermock-module-junit4:$rootProject.ext.powerMockVersion"
     testImplementation "org.powermock:powermock-module-junit4-rule:$rootProject.ext.powerMockVersion"
     testImplementation "org.powermock:powermock-api-mockito2:$rootProject.ext.powerMockVersion"
     testImplementation "org.powermock:powermock-classloading-xstream:$rootProject.ext.powerMockVersion"
     testImplementation "org.robolectric:robolectric:$rootProject.ext.robolectricVersion"
     testImplementation "androidx.test:core:$rootProject.ext.androidxTestCoreVersion"
     testImplementation project(":LabApiUtilities")
+    testImplementation project(":testutils")
```

Comment:
`testutils` internally depends on common...so if you use it in common then this would cause circular dependency. Please remove it

Reply:
Done

Reply:
Looks like this isn't actually addressed yet. I still see it here in the code.